.github/workflows/bundlers_tests.yml

```
@@ -16,8 +16,17 @@ jobs:
       with:
         node-version: '15'
 
+    - name: Restore npm cache
+      uses: actions/cache@v2
+      with:
+        path: ./node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-modules-${{ hashFiles('**/package-lock.json') }}
+          ${{ runner.os }}-node-modules
+
     - name: Run npm install
-      run: npm install --no-package-lock --no-audit --no-fund
+      run: npm ci --no-audit --no-fund
```

`npm ci` [will automatically remove](https://docs.npmjs.com/cli/v7/commands/npm-ci) the node_modules folder
```suggestion
      run: test -d node_modules || npm ci --no-audit --no-fund
```
The same for all other *.yml files.

---

I use the code from github actions documentation - https://github.com/actions/cache/blob/main/examples.md#macos-and-ubuntu